src/views/transactions/components/account-selector/account-selector.view.jsx

```diff
@@ -68,6 +68,9 @@ function AccountSelector({
                     </div>
                   );
                 } else {
+                  const accounts = accountsTask.data.accounts.filter(
+                    (account) => account.token.USD !== null
+                  );
```

Comment:
Since `token.USD` can not be `null` at the domain level, this filter makes no sense. We can do this now only because this view is not yet migrated to TS.